spot-contracts/contracts/FeePolicy.sol

```diff
@@ -274,14 +263,16 @@ contract FeePolicy is IFeePolicy, OwnableUpgradeable {
 
     /// @inheritdoc IFeePolicy
     function computePerpRolloverFeePerc(uint256 dr) external view override returns (int256) {
-        return
-            Sigmoid.compute(
-                dr.toInt256(),
-                perpRolloverFee.lower,
-                perpRolloverFee.upper,
-                perpRolloverFee.growth,
-                ONE.toInt256()
+        if (dr <= ONE) {
+            uint256 perpRate = MathUpgradeable.min(
```

Comment:
Let's call this something like `negPerpRate` (most accurate?) so it's more clear what this intermediate value is.

Then, maybe take the max of `-1* negPerpRate` and the debasement limit? Seems more in the spirit of the computation.